### PR TITLE
Crim update

### PR DESCRIPTION
**Pull request by Zeeroqt**
- Changed buying location for Red Phos
- Changed buying location for Cocaine
- Changed selling location for stolen goods
- Fix the ability to not buy crowbars for 2 different prices

### CODE REVIEW

File: resources/usa_stores/sv_generalstore.lua [hunk 1]
```diff
@@ -120,7 +119,6 @@ local HARDWARE_STORE_ITEMS = {
     { name = "Paint Remover", price = 1000, type = "misc", quantity = 1, legality = "legal", weight = 15, objectModel = "bkr_prop_meth_ammonia", remainingUses = 5, notStackable = true},
     { name = "Rag", price = 200, type = "misc", quantity = 1, legality = "legal", weight = 5, objectModel = "prop_shower_towel"},
     { name = "Sign Kit", price = 200, type = "misc", quantity = 1, legality = "legal", weight = 5, objectModel = "prop_cd_paper_pile1"},
-    { name = "Drill", price = 2000, legality = "legal", quantity = 1, type = "misc", weight = 10, objectModel = "hei_prop_heist_drill" },
```
Comment (minipunch) [inline resources/usa_stores/sv_generalstore.lua:123]: Why u remove the drill?

Comment (Zeeroqt) [reply resources/usa_stores/sv_generalstore.lua:123]: i didnt even realize i done that sorry!

**Comment by minipunch** [comment]
It would be good to get some sort of hint system in place for this because how tf are people supposed to know where to find these spots if they don't have insider information

**Comment by Zeeroqt** [comment]
> It would be good to get some sort of hint system in place for this because how tf are people supposed to know where to find these spots if they don't have insider information

tbh most of the time people grind to find things yk? thats how most servers work.

**Review by minipunch — APPROVED** [review]

**Pull request merged**